### PR TITLE
Update to remove tslint from cryptography - Partially Closes #4696

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR partially resolves #4696 

### How was it solved?

- Remove tslint from lisk-cryptography

### How was it tested?
- All the linting should work
- no source code changes

### CODE REVIEW

File: .eslintrc.test.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ module.exports = {
 		'arrow-body-style': 'off',
 		'@typescript-eslint/no-magic-numbers': 'off',
 		'@typescript-eslint/no-require-imports': 'off',
-		'@typescript-eslint/no-unsafe-member-access': 'off',
 		'@typescript-eslint/no-explicit-any': 'off',
+		'@typescript-eslint/no-unsafe-assignment': 'off',
+		'@typescript-eslint/no-unsafe-member-access': 'off',
```
Comment (nazarhussain) [inline .eslintrc.test.js:9]: This rule was already added to same file in the previous PR.

File: elements/lisk-cryptography/src/buffer.ts [hunk 2]
```diff
@@ -37,20 +37,23 @@ export const intToBuffer = (
...
 		} else {
+			// eslint-disable-next-line no-lonely-if
 			if (signed) {
 				buffer.writeBigInt64BE(BigInt(value));
 			} else {
 				buffer.writeBigUInt64BE(BigInt(value));
 			}
 		}
 	} else {
+		// eslint-disable-next-line no-lonely-if
```
Comment (nazarhussain) [inline elements/lisk-cryptography/src/buffer.ts:48]: At what point we will update the code based on new eslint rules. I suggest for such small things we should fix them right away.

File: elements/lisk-cryptography/src/hash.ts [hunk 3]
```diff
@@ -41,12 +41,13 @@ export const hash = (data: Buffer | string, format?: string): Buffer => {
 	}
 
 	throw new Error(
-		`Unsupported data:${data} and format:${format}. Currently only Buffers or hex and utf8 strings are supported.`,
+		`Unsupported data:${data} and format:${format ??
```
Comment (nazarhussain) [inline elements/lisk-cryptography/src/hash.ts:44]: For my knowledge what's the difference between: 

```
format ?? 'undefined' 
```

and 

```
format || 'undefined' 
```

Comment (shuse2) [reply elements/lisk-cryptography/src/hash.ts:44]: `??` will only check null or undefined, but || is falsy check

File: elements/lisk-cryptography/src/buffer.ts (hunk 2 above)
Comment (shuse2) [reply elements/lisk-cryptography/src/buffer.ts:48]: I was thinking to fix this, but it just made it hard to read, so i wasn't sure how to make it clearer

File: framework/src/controller/bus.js [hunk 4]
```diff
@@ -228,7 +228,7 @@ class Bus extends EventEmitter2 {
 	}
 
 	// eslint-disable-next-line @typescript-eslint/require-await
-	async leanup() {
+	async cleanup() {
```
Comment (mitsuaki-u) [inline framework/src/controller/bus.js:231]: 👍

File: .eslintrc.test.js (hunk 1 above)
Comment (shuse2) [reply .eslintrc.test.js:9]: target branch is up to date with the latest development branch?

File: elements/lisk-cryptography/src/buffer.ts (hunk 2 above)
Comment (nazarhussain) [reply elements/lisk-cryptography/src/buffer.ts:48]: We should then create a separate issue saying remove these comments and fix the code.

Comment (shuse2) [reply elements/lisk-cryptography/src/buffer.ts:48]: this function will be removed with serialization =)

**Pull request merged**